### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by pengin7384**
TODO
- Resolve ainize error

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM node:10
```
Comment (minsulee2) [inline Dockerfile:1]: Can you upgrade it as node:12?

File: README.md [hunk 2]
```diff
@@ -13,10 +13,11 @@ npm start
```
Comment (minsulee2) [inline README.md:13]: Please update it all from top to bottom.

File: Dockerfile (hunk 1 above)
Comment (pengin7384) [reply Dockerfile:1]: OK👌

File: README.md (hunk 2 above)
Comment (pengin7384) [reply README.md:25]: Update completed! Thx

**Pull request merged**